fuel-tests/tests/debugger.rs

```diff
@@ -0,0 +1,40 @@
+#![cfg(feature = "debug")]
+
+use fuel_core::service::{Config, FuelService};
+use fuel_gql_client::client::FuelClient;
+use fuel_vm::prelude::*;
+
+#[tokio::test]
+async fn debugger() {
```

Comment:
good sanity check, but would be helpful to have some comments or more descriptive test names to distinguish where new cases should be made vs added onto this test.

Reply:
I've clarified this a bit